pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -464,6 +467,21 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	// needed for luksOpen synchronization when devices are encrypted and the osd is prepared with LVM
 	hostIPC := osdProps.storeConfig.EncryptedDevice || osdProps.encrypted
 
+	osdLabels := c.getOSDLabels(osd, failureDomainValue, osdProps.portable)
+
+	if c.spec.Network.MultiClusterService.Enabled {
+		externalIP, err := c.createOSDService(osd, osdLabels)
+		if err != nil {
+			return nil, errors.Errorf("failed to configure osd service for osd.%d. %v", osd.ID, err)
+		} else {
+			args = append(args, []string{
+				fmt.Sprintf("--public-addr=%s", externalIP),
+				fmt.Sprintf("--public-bind-addr=$(ROOK_POD_IP)"),
+				fmt.Sprintf("--cluster-addr=$(ROOK_POD_IP)"),
```

Comment:
What is ROOK_POD_IP referring to here? Is it an operator pod or OSD POD?

Reply:
It would be the OSD pod IP